model/pdata/metrics.go

```diff
@@ -247,6 +247,42 @@ func (at MetricAggregationTemporality) String() string {
 	return otlpmetrics.AggregationTemporality(at).String()
 }
 
+// MetricDataPointFlags defines how a metric aggregator reports aggregated values.
+// It describes how those values relate to the time interval over which they are aggregated.
+type MetricDataPointFlags uint32
+
+const (
+	// MetricDataPointFlagsNone is the default MetricDataPointFlags.
+	MetricDataPointFlagsNone = MetricDataPointFlags(otlpmetrics.DataPointFlags_FLAG_NONE)
+)
+
+// NewMetricDataPointFlags returns a new MetricDataPointFlags combining the flags passed
+// in as parameters.
+func NewMetricDataPointFlags(flags ...MetricDataPointFlag) MetricDataPointFlags {
+	var flag MetricDataPointFlags
+	for _, f := range flags {
+		flag |= MetricDataPointFlags(f)
+	}
+	return flag
+}
+
+// HasFlag returns true if the MetricDataPointFlags contains the specified flag
+func (d MetricDataPointFlags) HasFlag(flag MetricDataPointFlag) bool {
+	return d&MetricDataPointFlags(flag) != 0
+}
+
+// String returns the string representation of the MetricDataPointFlags.
+func (d MetricDataPointFlags) String() string {
+	return otlpmetrics.DataPointFlags(d).String()
+}
+
+type MetricDataPointFlag uint32
```

Comment:
Please comment here, and signal the different type from Flags.

Reply:
Updated name to `MetricDataPointFlagValue` and added comment.